drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -947,8 +947,10 @@ static void device_run(void *priv)
 static int vidioc_querycap(struct file *file, void *priv,
 			   struct v4l2_capability *cap)
 {
+	struct bcm2835_codec_dev *dev = video_drvdata(file);
+
 	strncpy(cap->driver, MEM2MEM_NAME, sizeof(cap->driver) - 1);
-	strncpy(cap->card, MEM2MEM_NAME, sizeof(cap->card) - 1);
+	strncpy(cap->card, dev->vfd.name, sizeof(cap->card) - 1);
```

Comment:
You use strscpy in a later commit, which seems preferable, but strncpy here is consistent with the previous line. Your call.

Reply:
Indeed, it's just that. I'm not adding or changing the strncpy. I'm only changing the MEM2MEM_NAME parameter.

If you'd like the strncpy changed to strscpy, then that should be a separate patch.